src/main/java/com/google/devtools/build/lib/remote/util/DigestUtil.java

```diff
@@ -164,6 +164,11 @@ public static String toString(Digest digest) {
     return digest.getHash() + "/" + digest.getSizeBytes();
   }
 
+  public static Digest fromString(String digest) {
+    String[] parts = digest.split("/");
+    return buildDigest(parts[0], Long.parseUnsignedLong(parts[1]));
```

Comment:
Check that there's only two parts? And why unsigned, given `toString()` above uses regular string conversion?